fonts.css

```diff
@@ -0,0 +1,11 @@
+@import url('https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap');
+
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap');
+
+* {
+	font-family: 'Nanum Gothic', sans-serif;
+}
+
+h1, h2, h3, h4 {
+	font-family: 'Josefin Sans', sans-serif;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/FW/pullRequest?prid=5459839)